### PR TITLE
Diagnostic study

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-export/catI-r5/qrda_templates/diagnostic_study_performed.mustache [hunk 1]
```diff
@@ -10,20 +10,41 @@
...
+      <!-- QDM Attribute: Method -->
+      {{> qrda_templates/template_partials/_method}}
+    {{/method}}
     {{#authorDatetime}}
-    {{> qrda_templates/template_partials/_author}}
+      <!-- QDM Attribute: Author dateTime -->
+      {{> qrda_templates/template_partials/_author}}
     {{/authorDatetime}}
+    {{#facilityLocation}}
+      <!-- QDM Attribute: Facility Locations -->
```
Comment (mayerm94) [inline lib/qrda-export/catI-r5/qrda_templates/diagnostic_study_performed.mustache]:
```suggestion
      <!-- QDM Attribute: Facility Location -->
```
